content/maxwell1_fundamentals/plane_waves_in_homogeneous_media/index.rst

```diff
@@ -64,67 +57,94 @@ Analytic solution exists for this simple setup, and we first obtain solution in
 
 .. math::
 
     \boldsymbol{\nabla}^2 \mathbf{h} - \mu\epsilon \frac{\partial^2 \mathbf{h}}{\partial t^2} - \mu\sigma \frac{\partial \mathbf{h}}{\partial t}    = 0,
 
-with an initial condition:
+with the initial condition:
 
 .. math::
     \mathbf{h}(t=0)=\delta(t)\mathbf{H}_0, \ \text{when} \ z=0,
 
     \mathbf{h}(t=0)=0, \ \text{when} \ z\neq 0.
 
-Physical meaning of this case could be a lightening, which would be an impulse current resulting in impulse magnetic field at a specific point from Ampere's law. The other case where impulse electric field exists can correspond to a long-lasting lightening then abruptly shut-off (like step function), which can be inferred from Faraday's law.
-
 .. note::
+    Although solutions to the Helmholtz equation contain terms of the form :math:`e^{-i\omega t}`, these terms are often suppressed for more concise notation.
 
-    Time derivative of step function :math:`u(t)` is :math:`\delta(t)`.
+**Summary:**
 
-On our following derivation we focus on solving Maxwell's equation for the electric field with b) in both frequency and time domain.
+Solutions to the Helmholtz equations, and the wave equations, can be obtained two ways. Either we solve for the electric field and use it to obtain the magnetic field, or visa versa. In the following material, we will solve the scenario illustrated in :numref:`planewavedown` by using our first choice in boundary conditions; which solves for the electric field and uses it to obtain the magnetic field.
 
+In following sections, we derive analytic expressions for the plane wave EM fields in both frequency and time domain. Important physical behaviours of each solution are discussed (Analytic Solution section) along with the relationship between EM fields (Fields section). Finally, using numerical apps, we simulate plane EM fields, and visualize them (Simulation section).
+
+.. _plane_waves_in_homogeneous_media_index_Poynting:
+
+Poynting Vector
+---------------
+
+The Poynting vector (:math:`\mathbf{S}`) defines the directional energy flux density of an electromagnetic field. The Poynting vector is useful from a qualitative standpoint because it can be used to relate the electric field, magnetic field and propagation direction of EM waves. This quantity is given by the following equation:
+
+.. math::
+    \mathbf{S} = \mathbf{E \times H}
+    :name:
 
-In following sections, we derive analytic expressions for the plane EM fields in both frequency and time domain, understand important physical behavior of them (Analytic Solution section), discuss relationship between EM fields (Fields section).
-Further, using plane wave apps, we simulate plane EM fields, and visualize them (Simulation section).
+Thus if an EM wave at a certain time and location were to support an electric field in the positive x-direction and a magnetic field in the negative y-direction, we could deduce that the EM wave is propagating downward.
 
-**Contents**
+Contents
+--------
```

Comment:
The navigation here is a bit tricky (the "Contents" and having titles with brackets is potentially quite confusing). 
![image](https://cloud.githubusercontent.com/assets/6361812/24690173/0bc47132-1980-11e7-93fc-caa72ebad6a7.png)

I would instead suggest:
- Plane Waves in Homogeneous Media
    - Intro
    - setup
    - solving
    - frequency domain (show the Analytic solution on the index page)
        -  fields
        - analytic solution
    - time domain (show the Analytic solution on the index page)
        -  fields
        - analytic solution
     
 cc @sgkang, @dougoldenburg